Formula/imagemagick@6.rb

```diff
@@ -0,0 +1,148 @@
+  depends_on "fontconfig" => :optional
+  depends_on "little-cms" => :optional
+  depends_on "little-cms2" => :optional
+  depends_on "libwmf" => :optional
+  depends_on "librsvg" => :optional
+  depends_on "liblqr" => :optional
+  depends_on "openexr" => :optional
+  depends_on "ghostscript" => :optional
+  depends_on "webp" => :optional
+  depends_on "openjpeg" => :optional
+  depends_on "fftw" => :optional
+  depends_on "pango" => :optional
+  depends_on :perl => ["5.5", :optional]
+
+  needs :openmp if build.with? "openmp"
+
+  skip_clean :la
+
+  def install
+    args = %W[
+      --disable-osx-universal-binary
+      --prefix=#{prefix}
+      --disable-dependency-tracking
+      --disable-silent-rules
+      --enable-shared
+      --enable-static
+    ]
+
+    if build.without? "modules"
+      args << "--without-modules"
+    else
+      args << "--with-modules"
+    end
+
+    if build.with? "opencl"
+      args << "--enable-opencl"
+    else
+      args << "--disable-opencl"
+    end
+
+    if build.with? "openmp"
+      args << "--enable-openmp"
+    else
+      args << "--disable-openmp"
+    end
+
+    if build.with? "webp"
+      args << "--with-webp=yes"
+    else
+      args << "--without-webp"
+    end
+
+    if build.with? "openjpeg"
+      args << "--with-openjp2"
+    else
+      args << "--without-openjp2"
+    end
+
+    args << "--without-gslib" if build.without? "ghostscript"
+    args << "--with-perl" << "--with-perl-options='PREFIX=#{prefix}'" if build.with? "perl"
+    args << "--with-gs-font-dir=#{HOMEBREW_PREFIX}/share/ghostscript/fonts" if build.without? "ghostscript"
+    args << "--without-magick-plus-plus" if build.without? "magick-plus-plus"
+    args << "--enable-hdri=yes" if build.with? "hdri"
+    args << "--without-fftw" if build.without? "fftw"
+    args << "--without-pango" if build.without? "pango"
+    args << "--without-threads" if build.without? "threads"
+    args << "--with-rsvg" if build.with? "librsvg"
+    args << "--without-x" if build.without? "x11"
+    args << "--with-fontconfig=yes" if build.with? "fontconfig"
+    args << "--with-freetype=yes" if build.with? "freetype"
+    args << "--enable-zero-configuration" if build.with? "zero-configuration"
+
+    if build.with? "quantum-depth-32"
+      quantum_depth = 32
+    elsif build.with?("quantum-depth-16") || build.with?("perl")
+      quantum_depth = 16
+    elsif build.with? "quantum-depth-8"
+      quantum_depth = 8
+    end
+    args << "--with-quantum-depth=#{quantum_depth}" if quantum_depth
+
+    # versioned stuff in main tree is pointless for us
+    inreplace "configure", "${PACKAGE_NAME}-${PACKAGE_VERSION}", "${PACKAGE_NAME}"
+    system "./configure", *args
+    system "make", "install"
+  end
+
+  def caveats
+    s = <<-EOS.undent
+      For full Perl support you may need to adjust your PERL5LIB variable:
+        export PERL5LIB="#{HOMEBREW_PREFIX}/lib/perl5/site_perl":$PERL5LIB
+    EOS
+    s if build.with? "perl"
+  end
```

Comment:
Probably just worth removing these caveats (or adjusting them to point to `opt_lib` )